src/openrct2/windows/options.c

```diff
@@ -439,7 +445,7 @@ static rct_window_event_list window_options_events = {
 	(1 << WIDX_TAB_4) | \
 	(1 << WIDX_TAB_5) | \
 	(1 << WIDX_TAB_6) | \
-	(1 << WIDX_TAB_7)
+	TWITCH_TAB_ENABLED
```

Comment:
Is this necessary? I can't remember if setting `disabled_widgets` will prevent its click handler - I would of thought so.